src/structures/ClientUser.js

```diff
@@ -82,7 +82,13 @@ class ClientUser extends User {
   edit(data, password) {
     const _data = {};
     _data.username = data.username || this.username;
-    _data.avatar = this.client.resolver.resolveBase64(data.avatar) || this.avatar;
```

Comment:
All you really need to do is change `|| this.avatar` to `|| Buffer.alloc(0)` here and remove everything else you added.
This would allow you to do `.setAvatar('')` and `.setAvatar(null)`